VisualStudio.gitignore

```diff
@@ -183,6 +183,7 @@ AppPackages/
 BundleArtifacts/
 Package.StoreAssociation.xml
 _pkginfo.txt
+.appx
```

Comment:
Indeed. When I create packages for the Windows Store, I choose a different folder than the `AppPackages/` folder (step 6 [here](https://docs.microsoft.com/en-us/windows/uwp/packaging/packaging-uwp-apps#create-an-app-package)
). Anyway, some .appx files are still in the project root folder, but I don't know if this is caused by me choosing a different folder for the Store packages. 
![2017-05-12 2](https://cloud.githubusercontent.com/assets/20776298/25983910/373f3cf6-36f0-11e7-8612-47d20b8715e1.png)